changelog/1.28.0.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "1.28.0"
+description: "Released on 02/16/2022"
+---
+
+### Breaking changes ❗
+
+There are no breaking changes in 1.28.0.
```

Comment:
can we add a note in here, we now require Kubernetes 1.21 for 1.28: https://github.com/coder/enterprise-helm/pull/215

I can open a PR if you'd prefer

```
### Breaking changes ❗

- infra: Coder now requires the use of Kubernetes v1.21 or later. See Coder's
  [version support policy] for more information.

<!-- Turn off linting to avoid changing the link -->
<!-- markdownlint-disable MD044 -->

[version support policy]:
  ../setup/kubernetes/index.md#supported-kubernetes-versions

<!-- markdownlint-enable MD044 -->
```